reference/forms/types/textarea.rst

```diff
@@ -28,6 +28,13 @@ Renders a ``textarea`` HTML element.
 | Class       | :class:`Symfony\\Component\\Form\\Extension\\Core\\Type\\TextareaType` |
 +-------------+------------------------------------------------------------------------+
 
+.. tip::
+
+    If you prefer to use an **advanced WYSIWYG editor** instead of a plain
+    textarea, consider using the `IvoryCKEditorBundle`_ community bundle. Read
+    `its documentation </doc/current/bundles/IvoryCKEditorBundle/index.html>`_
```

Comment:
It's correct. However, we always use absolute URLs (including domain name) and we don't inline the links. Can you please update it, @javiereguiluz ?